Lighthouse Camera/UI/Main/ViewController.swift

```diff
@@ -111,6 +110,9 @@ class ViewController: UIViewController {
     self.isBusy = false;
     sViewController = self;
 
+    // Initialize the delegate to self indicating viewcontroller is going to implemenent PreviewViewDelegate
+    self.previewView.delegate = self
```

Comment:
Explicitly declaring delegate variable inside PreviewView class to be weak so that it holds weak reference to view controller and will be deallocated  See https://www.natashatherobot.com/ios-weak-delegates-swift/